pkg/providers/cloudstack/decoder/decoder.go

```diff
@@ -52,11 +52,20 @@ func ParseCloudStackSecret() (*CloudStackExecConfig, error) {
 			return nil, fmt.Errorf("'verify-ssl' has invalid boolean string %s: %v", verifySslValue, err)
 		}
 	}
+	timeout, err := section.GetKey("timeout")
```

Comment:
I don't think this is the right place to put this field. It seems like this should be an independent environment variable, not encoded in the b64 exec config string. This encoded string will be passed to CAPC directly and it doesn't make sense to include this variable there in my opinion.

Reply:
The only inputs for using cloudstack with eks-a are this b64-encoded ini file, which contains the cloudstack management url, api key, secret key, and verify-ssl boolean, and then the eks-a cluster spec. I was proposing we introduce a new environment variable for overriding the timeout since that only applies to the preflight check, whereas Wonkun's initial implementation had it as part of the b64-encoded input ini file